Tests/Resources/app/config/cmf_seo.yml

```diff
@@ -14,7 +14,13 @@ cmf_seo:
     error:
         enable_parent_provider: true
         enable_sibling_provider: true
-    sitemap: true
+    sitemap:
+        configurations:
+            default:
+                default_change_frequency: never
+                templates:
+                    xml: 'CmfSeoBundle:Sitemap:index.xml.twig'
+                    html: 'CmfSeoBundle:Sitemap:index.html.twig'
```

Comment:
imho we should not need this explicitly when its the default values. we should just say sitemap: true and the default sitemap with these defaults is automatically defined. the Configuration could auto-add things. 

i can do a PR tomorrow to do this if you want.